golang.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "context"
+
+//here is a line coment
+
+// FileGetter is used to retriv the code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
+	GetFiles(context.Context, *FilesRequest) (FileScanner, error)
+}
+
+func main() {
+	/*
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:comment (48),cement (41),moment (38)